### PR TITLE
Chore: Fix duplicate imports

### PR DESCRIPTION
**Pull request by thisisobate**
**What this PR does / why we need it**:
Fixes duplicate imports
**Which issue(s) this PR fixes**:
Ref to PR [#30989](https://github.com/grafana/grafana/pull/30989)

### CODE REVIEW

**Review by torkelo — APPROVED** [review]
Looks good!

**Review by dprokop — COMMENTED** [review]
Looks good @thisisobate. Left few suggestions. Instead of merging this to master I suggest merging this into [my branch](https://github.com/grafana/grafana/pull/30989) first so that we avoid merging red PRs:P 

To do that you need to:
1. [Rebase](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) your branch into mine(`eslint-no-duplicate-imports`) and force push it (git push --force)
2.  **Change the target of this PR** to my branch. You do it by editing the PR:   
  image


Alternatively, you can merge this first, then syncc my branch with master and merge it:) But maybe a quick lesson in rebasing will be useful :)

**Comment by thisisobate** [comment]
> Rebase your branch into mine(eslint-no-duplicate-imports)
Isn't this supposed to be the other way round?
Like,  `Rebase your branch(eslint-no-duplicate-imports into mine(remove-duplicate-imports)`?

File: public/app/features/plugins/plugin_loader.ts [hunk 1]
```diff
@@ -40,6 +40,7 @@ import impressionSrv from 'app/core/services/impression_srv';
 import builtInPlugins from './built_in_plugins';
 import * as d3 from 'd3';
 import * as emotion from 'emotion';
+// eslint-disable-next-line no-duplicate-imports
```
[the remark this answers is not in the document]
Comment (thisisobate) [reply public/app/features/plugins/plugin_loader.ts:43]: Yeah...that would work perfectly fine

**Comment by thisisobate** [comment]
> > Rebase your branch into mine(eslint-no-duplicate-imports)
> 
> Isn't this supposed to be the other way round?
> Like, Rebase your branch(eslint-no-duplicate-imports) into mine(remove-duplicate-imports) since I'll be squashing and merging to eslint-no-duplicate-imports branch? @dprokop

Oh...just figured.
You're right. I misunderstood your point.

English can be hard for me to understand sometimes 😅

**Pull request merged**